vNext/common/config/rush/command-line.json

```diff
@@ -0,0 +1,189 @@
+/**
+ * This configuration file defines custom commands for the "rush" command-line.
+ * For full documentation, please see https://rushjs.io
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json",
+
+  /**
```

Comment:
Can the comments be deleted? Is this file needed?

Reply:
Cleaned up comments. added support for test command.